src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table').children;
+
+const header = (table[0]);
+const tBody = (table[1]);
+const tFooter = (table[2]);
+
+function findPosition(element, index) {
+  const rows = element.children[index];
+  const insertPosition = element.children[index].children[4];
+  let copy = document.createElement('th');
+
+  if (element.children.length > 1) {
+    copy = document.createElement('td');
+  }
+
+  copy.innerText = rows.children[1].innerText;
+  element.children[index].insertBefore(copy, insertPosition);
+}
+
+for (let i = 0; i < tBody.children.length; i++) {
+  findPosition(tBody, i);
+};
+
+for (let i = 0; i < header.children.length; i++) {
+  findPosition(header, i);
+};
+
+for (let i = 0; i < tFooter.children.length; i++) {
```

Comment:
Use functional programming method `forEach` instead of `for` loop. For example, use `tFooter.querySelectorAll('tr').forEach((row, index) => { ... })`.